### ISSUE TITLE
Focus outline on tabs hard to see

### ISSUE THREAD

**Issue by TimJentzsch** [MurdoMaclachlan/website#36]
This is just a minor issue, the navigation works well now!

When the tabs are focused, the focus outline on the top and bottom is not visible (on Firefox):
![image](https://user-images.githubusercontent.com/13908946/205990839-948766b3-932f-4b75-9f02-f9478ab2582b.png)

Adding a `margin` to the `ul` element that contains all tabs seems to fix it (I tried `0.5em` but you can play around with it)
